docs/translation.md

```diff
@@ -29,16 +25,16 @@ Create `~/.transifexrc` in the following format and insert the API token:
 
 Push to Transifex
         
-    tx push --source --translations
+    ./manage.py push_messages
 
 Wait for strings to be translated
 
 Pull from Transifex
 
-    tx pull --force --language=cy
+    ./manage.py pull_messages
         
 Compile the translations
 
-    pybabel compile -d cla_public/translations
+    pybabel compile -f -d cla_public/translations
```

Comment:
Do we need to update these in the `Dockerfile`?

Currently, it's https://github.com/ministryofjustice/cla_public/blob/9c0b007a9b1c2111a75f45182b2fc94345ac13aa/Dockerfile#L76